bpfman/src/lib.rs

```diff
@@ -12,7 +12,7 @@ use aya::{
         fentry::FEntryLink, fexit::FExitLink, kprobe::KProbeLink, links::FdLink, loaded_programs,
         trace_point::TracePointLink, uprobe::UProbeLink, FEntry, FExit, KProbe, TracePoint, UProbe,
     },
-    BpfLoader, Btf,
+    Btf, EbpfLoader,
```

Comment:
is the order change here expected or that is a typo ?

Reply:
Yes the old Bpf types are marked as deprecated and will be removed in a future release